components/aws/sagemaker/tests/integration_tests/utils/ack_utils.py

```diff
@@ -16,31 +16,71 @@ def _get_resource(k8s_client, job_name, kvars):
     _api = client.CustomObjectsApi(k8s_client)
     namespace = os.environ.get("NAMESPACE")
     job_description = _api.get_namespaced_custom_object(
-        kvars["group"].lower(),
-        kvars["version"].lower(),
+        "sagemaker.services.k8s.aws",
+        "v1alpha1",
```

Comment:
Could the version change in the future?

Reply:
Unlikely, mainly because the crds often change per ack controller release which makes it more convenient to use v1alpha1.